x/genmsg/integration_test.go

```diff
@@ -0,0 +1,27 @@
+package genmsg_test
+
+import (
+	"testing"
+)
+
+func TestIntegration(t *testing.T) {
+	//recvAddr := sdk.AccAddress("recv")
```

Comment:
You can use the helper in `common/testutil/cli` with the custom genesis, which will test your `InitGenesis` function. It's an easy way to create a happy path test for the whole module.